docs/_sass/_docs-display-component.scss

```diff
@@ -14,6 +14,7 @@
     transition: background-color .25s;
 
     > .fd-tile__content {
+        // width: 100%;
```

Comment:
^ Don't forget to delete if it's no longer needed 👍

Reply:
removed thanks :)